app/controllers/posts_controller.rb

```diff
@@ -10,10 +10,13 @@ def new
 
   def create
```

Comment:
Please solve the `n+1` problem in `posts#index` using eager loading as it was requested by the previous code reviewer.

Reply:
As for this part, bullet is suggesting against it. Please `bundle install`, view any user's profile and  try clicking on `see all posts`. Try introducing this change.